editor/editor_node.cpp

```diff
@@ -5137,16 +5137,36 @@ void EditorNode::_scene_tab_closed(int p_tab) {
 	scene_tabs->update_scene_tabs();
 }
 
-Button *EditorNode::add_bottom_panel_item(String p_text, Control *p_item) {
-	Button *tb = memnew(Button);
+class EditorBottomDockButton : public Button {
```

Comment:
You could use drag forwarding instead of adding a new class.

Reply:
Seemed more hacky to me, I think the new class is simpler.

Reply:
How is adding a method more hacky than a mini-class defined in cpp file 🙃

Reply:
I guess just personal preference